Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -179,7 +179,17 @@ class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: ObservableObj
                     livenessImages: livenessFiles
                 )
                 allFiles.append(info)
-                let zipData = try LocalStorage.zipFiles(at: allFiles)
+                let zipData: Data
+                do {
+                    let securityInfo = try createSecurityInfo(files: allFiles)
+                    zipData = try LocalStorage.zipFiles(
+                        urls: allFiles,
+                        data: ["security_info.json": securityInfo]
+                    )
+                } catch {
+                    /* in case we can't add the security info the backend will deal with the enrollment */
```

Comment:
Same here. We are not catching any errors.

Reply:
Resolved in [f1c0a73](https://github.com/smileidentity/ios/pull/286/commits/f1c0a7342480f07515b317fc20349c7a7ddf0bd5)